common/scripted_triggers/wc_his_character_triggers.txt

```diff
@@ -164,6 +212,10 @@ is_any_historical_character_trigger = {
 		is_thoras_trigger = yes
 		is_varian_trigger = yes
 		is_katherine_proudmoore_trigger = yes
+		is_tiffin_trigger = yes
+		is_alexandros_mograine_trigger = yes
```

Comment:
You created a trigger named `is_alexandros_trigger`, not `is_alexandros_mograine_trigger`.